local-process-quickstart.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash cloudshell for a seamless experience.
+#1. az
```

Comment:
Please consider providing a download link, and mention that this is the "Azure CLI".

Reply:
Done!